### PR TITLE
Localizes content/en/docs/tutorials/kubernetes-basics/update/update-interactive.html to bengali

### PR DESCRIPTION
**Pull request by PratikforCoding**
Signed-off by: PratikforCoding kotalpratik@gmail.com
Localizes **content/en/docs/tutorials/kubernetes-basics/update/update-interactive.html**
Tracking issue: #31677

### CODE REVIEW

File: content/bn/docs/tutorials/kubernetes-basics/update/update-interactive.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    <main class="content katacoda-content">
+
+        <div class="katacoda">
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/6" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+               <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/scale/scale-interactive/" role="button">&lt; Return to Module 5<span class=""></span></a>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/update/update-interactive.html:24]: Kindly localize this part ""Return to Module 5"

File: content/bn/docs/tutorials/kubernetes-basics/update/update-interactive.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+               <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">Return to Kubernetes Basics<span class=""></span></a>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/update/update-interactive.html:25]: Kindly localize this part "Return to Kubernetes Basics"

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by mitul3737** [comment]
@PratikforCoding Kindly resolve the issues or I will close this PR within the next 1 week.

**Comment by mitul3737** [comment]
@PratikforCoding  kindly sign EasyCLA. 

You can find about EasyCLA here - https://docs.linuxfoundation.org/lfx/easycla

and the steps to sign EasyCLA are here - https://docs.linuxfoundation.org/v2/easycla/contributors/individual-contributor

You can start your signing EasyCLA from here - https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/51478266/38056/#/?version=2

File: content/bn/docs/tutorials/kubernetes-basics/update/update-interactive.html (hunk 2 above)
Comment (PratikforCoding) [reply content/bn/docs/tutorials/kubernetes-basics/update/update-interactive.html]: Done them, Thanks for the review!!

**Review by mitul3737 — APPROVED** [review]

**Pull request merged**